### DISCUSSION TITLE
Welcome to royal-diadems-blog Discussions!

### DISCUSSION THREAD

**Discussion by d-bitok** [d-bitok/royal-diadems-blog#18] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d-bitok** [comment]
Changing the landing page to show the crousel

**Reply by Pheon-Dev** [reply]
Only works with images when using mantine

**Comment by Pheon-Dev** [comment, marked as answer]
Try htmx

**Reply by d-bitok** [reply]
Sure will do

**Comment by d-bitok** [comment]


**Reply by Pheon-Dev** [reply]
Next js and supabase
